### PR TITLE
Feature/initial versioning

### PR DESCRIPTION
**Pull request by jordanbinskin**
Removing some of our initial files that are redundant and adding in a version of the project json and an example component.

### CODE REVIEW

File: json/components/funky-nav.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      "height": ""
+    },
+    "allowCalculateOverride": false,
+    "encrypted": false,
+    "showCharCount": false,
+    "showWordCount": false,
+    "properties": {},
+    "allowMultipleMasks": false,
+    "id": "eoltx77"
+  }
```
Comment (DanielTurner) [inline json/components/funky-nav.json:73]: Github is asking for an extra line at the end. Is that valid json if you add it?

File: json/project.json [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+          "key": "submit",
+          "tableView": false,
+          "label": "Submit",
+          "input": true
+        }
+      ]
+    }
+  },
+  "excludeAccess": true
+}
```
Comment (DanielTurner) [inline json/project.json:302]: Same as before.

**Review by DanielTurner — APPROVED** [review]

File: json/components/funky-nav.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
+{
+  "label": "HTML",
+  "tag": "nav",
+  "className": "funky",
```
Comment (nishgakkhar) [inline json/components/funky-nav.json:4]: Why is that called funky?

Comment (jordanbinskin) [reply json/components/funky-nav.json:4]: This is for an example component, not intend to be re-used.

**Review by nishgakkhar — APPROVED** [review]
All good..

**Pull request merged**